app/core/models/ocelot/errors/GuidanceErrors.scala

```diff
@@ -46,6 +46,8 @@ case class FlowParseError(id: String, msg: String, arg: String) extends FlowErro
 case class MetaParseError(id: String, msg: String, arg: String) extends MetaError
 case class PhrasesParseError(id: String, msg: String, arg: String) extends PhrasesError
 case class LinksParseError(id: String, msg: String, arg: String) extends LinksError
+case class MultipleExclusiveOptionsError(id: String) extends FlowError
+case class MissingNonExclusiveOptionError(id: String) extends FlowError
```

Comment:
if we validate the incoming Sequence, dont think this error can happen

Reply:
MissingNonExclusiveOptionError has been removed as it is redundant. However, there is nothing to stop a designer putting [exclusive] into the text for two options so multiple exclusive options is still required.